remove user schema from the column search

kimakan opened the pull request:
Closes #178

agy-why reviewed (approved):
all fine as discussed

kimakan commented:
I have simplified and fixed the control for the rendering of the tooltips. Previously, the tooltips stayed rendered forever sometimes due to the re-rendering of the user_schema. In the old solution, I just turned off the tooltips for the user_schema but it's not a nice way of doing it. Now, the tooltips are shown for every schema, table and column. I ensure that no tooltip stays rendered forever by hiding all tooltips on the webpage after the mouseover event is over.
